routes/user.js

```diff
@@ -89,5 +94,153 @@ module.exports = [
         reply(Boom.badImplementation(err));
       });
     }
+  },
+
+  {
+    method: 'GET',
+    path: '/user/{user}/catalog.json',
+    config: {
+      tags: ['disablePlugins']
+    },
+    handler: (request, reply) => {
+      const { user } = request.params;
+
+      return Promise.all([
+        User.findOne({
```

Comment:
Is the user record necessary for the catalog json?  Can we just use the user id then we can avoid the `findOne` query and just perform the `Meta.find`

Reply:
Just to fetch the username for descriptive purposes (which is never surfaced anywhere public).